_docs/2.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,142 @@
+
+`GET {menas_url}/api/{entity}/exportItem/{name}/{version}`
+
+- entity is `schema`, `dataset` or `mappingTable`
+- name is the name of the entity as seen in Menas
+- version is the version of the entity as seen in Menas. Version is optional and in that case the latest version will be used
+
+#### Example Import Response for Schema
+
+Returns code is 200.
+
+**Top level keys**
+
+| Key | Value | Description |
+|---|---|---|
+| metadata | Struct | Metadata for the export functionality |
+| item | Struct | exported entity |
+
+**Metadata keys**
+
+| Key | Value | Description |
+|---|---|---|
+| exportVersion | Integer | Version of the export and model used |
+
+**Item keys**
+
+| Key | Value | Description |
+|---|---|---|
+| name | String | the name of the entity |
+| description | String | the description of the entity |
+| fields | Struct | fields specified in the schema. A complex type representing columns. |
+
+**Fields keys**
+
+| Key | Value | Description |
+|---|---|---|
+| name | String | the name of the column |
+| type | String | the type of the column |
+| path | String | parent column name |
+| absolutePath | String | absolute path to the column |
+| elementType | String | if field is a struct or array |
+| nullable | Boolean | boolean value specifying nullability |
+| metadata | Struct | metadata of the field, like patters, default value, etc. |
+| children | Array of Fields | array of fields. Sub-columns/fields to this one. |
+
+
+Example:
+```json
+{
+  "metadata": {
+    "exportVersion": 1
+  },
+  "item": {
+    "name": "Name",
+    "description": "",
+    "fields": [
+      {
+        "name": "Boolean",
+        "type": "boolean",
+        "path": "",
+        "elementType": null,
+        "containsNull": null,
+        "nullable": true,
+        "metadata": {
+          "default": null
+        },
+        "children": [],
+        "absolutePath": "Boolean"
+      },
+      {
+        "name": "Byte",
+        "type": "byte",
+        "path": "",
+        "elementType": null,
+        "containsNull": null,
+        "nullable": true,
+        "metadata": {},
+        "children": [],
+        "absolutePath": "Byte"
+      }
+    ]
+  }
+}
+```
+
+### Entity Import
+
+This endpoint imports a single entity. All imports are versionless. If the import does not find an entity with the same name it will create a new one and start the version from 1. If the import finds an existing version, it will update the previous version.
+
+Versions of connected entities need to be specified properly. Export of a Dataset carries a Schema and maybe some Mapping tables as connected entities. These have versions and these versions need to exist on Import.
+
+`POST {menas_url}//api/{entity}/importItem`
+
+- entity is `schema`, `dataset` or `mappingTable`
+- expects a JSON payload
+
+#### JSON payload
+
+JSON payload is the same as the JSON response from the [export](#example-import-response-for-schema).
+
```

Comment:
| Code | Description |
|------|-------------|
| 201  | Successful import of the entity |
| ???  | Error, with payload describing the error |

Reply:
If there truly is a Location header (that 201 Created should be accompanied with), maybe there could be a mention of that, too?

Reply:
@dk1844 I do not know what is location header.